src/frequenz/sdk/microgrid/fuse.py

```diff
@@ -0,0 +1,19 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Fuse data class."""
+
+from dataclasses import dataclass
+
+from ..timeseries import Current
+
+
+@dataclass(frozen=True)
+class Fuse:
+    """Fuse data class.
+
+    Attributes:
+        max_current: Rated current of the fuse, in Amperes.
```

Comment:
As `max_current` is `Current` type we don't need to explicitly document the unit.
And we usually document attributes below the attribute definition as follows:

```python
@dataclass(frozen=True)
class Fuse:
    """Fuse data class."""

max_current: Current
"""Rated current of the fuse."""

Reply:
Done! Thanks for the suggestion. Do we want to enforce this style in doc tests?

Reply:
No, not really. Test might not need documentation except some special cases, it is up to the developer